xtext-website/_posts/releasenotes/2020-06-02-version-2-22-0.md

```diff
@@ -16,6 +16,8 @@ As you might have recognized, the number of people contributing to Xtext on a re
 
 The [Xtext community website](https://www.eclipse.org/Xtext/community.html) and the [Xtext documentation website](https://www.eclipse.org/Xtext/documentation/index.html) have been updated. New Xtext-based projects have been added, some editor features and the Unit testing / UI testing capabilities have been documented and the embedded code snippets have been updated as well.
 
+Furthermore, the [Xtext Developer Documentation](https://github.com/eclipse/xtext/tree/master/dev-doc) has been extended by a step-by-step introduction on [How to regenerate the test- and production Xtext languages](https://github.com/eclipse/xtext/blob/master/dev-doc/RegenerateLanguages.md) to enable the community to easily contribute to Xtext.
```

Comment:
have thought of more generalization here

Reply:
@cdietrich I updated the PR as requested.